recaf-ui/src/main/java/me/coley/recaf/presentation/GuiPresentation.java

```diff
@@ -49,6 +52,14 @@ public void initialize(Controller controller) {
 		// Setup listener to ensure we update classpath dependency directories
 		CompileDependencyUpdater.install(controller);
 		DecompileInterception.install(controller);
+        // Turn on the plugin that needs to be enabled
+		RecafPlugin.getInstance().enablePlugins(Configs.plugin().descriptor
+                .entrySet().stream()
+				.filter(Map.Entry::getValue)
+				.map(Map.Entry::getKey)
+				.collect(Collectors.toSet())
+        );
+		// PluginLoader.install(controller);
```

Comment:
I see this commented out, can probably move this stream-collect logic to a class like this though for brevity here.

Reply:
The `PluginLoader` I designed at the beginning, I forgot to comment it out for some reason, I will modify this part of the code.